prowler/providers/azure/azure_provider.py

```diff
@@ -79,38 +85,91 @@ def __init__(self, arguments):
 
     @property
     def identity(self):
+        """
```

Comment:
Please follow the Google comment format -> https://github.com/google/styleguide/blob/gh-pages/pyguide.md#383-functions-and-methods.

This applies to all the new method docs.

Reply:
Done